plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/MavenFacetImpl.java

```diff
@@ -469,7 +475,7 @@ public Boolean execute(final StorageTx tx) {
 
       @Override
       public String toString() {
-        return "setLastVerified(" + path.getPath() + ", " + verified + ")";
+        return SimpleFormat.format("setLastVerified(%s)", path.getPath(), verified);
```

Comment:
FTR most IDE understand String.format and likely point out missing tokens... where SimpleFormat.format() unless you train the IDE about non-standard replacement of String.format() will not.